inc/namespace.php

```diff
@@ -221,20 +221,29 @@ function enqueue_scripts() {
 	 */
 	$noop = (bool) apply_filters( 'altis.analytics.noop', false );
 
+	/**
+	 * Filters the consent cookie prefix to integrate with the WordPress Consent API.
+	 *
+	 * @param string|null $cookie_prefix The consent cookie prefix or false to skip consent check.
+	 */
+	$consent_cookie = apply_filters( 'altis.analytics.consent_cookie_prefix', null );
```

Comment:
I would use the default `_altis_consent` rather than `null` here. I wouldn't want the default (null) to be passed through and then not detect the cookies (but maybe I'm mis-reading this?).

Reply:
Ah sorry the intention is for this to work as before but requires you to filter the value elsewhere so it's not directly dependent on the Altis cookie consent plugin. It would work with any implementation of the WP Consent API this way. Maybe I'm overthinking that but I want to keep it somewhat capable of working outside of the Altis context.

Reply:
If you want it to work outside of the Altis context, you could use the WP-Consent-API filter for this that we wrap around, which is `wp_consent_cookie_prefix`. Then it could work either way.

Reply:
Updated to use this 👍